src/selectors/entities/preferences.ts

```diff
@@ -174,7 +175,13 @@ export const getTheme: (state: GlobalState) => Theme = createShallowSelector(
             theme.mentionBg = theme.mentionBj;
         }
 
-        return Object.assign({}, defaultTheme, theme);
+        return {
```

Comment:
We don't have to do this now, but we did talk about having this mapping put in a different function so that the mapping logic wasn't directly mixed with the logic for getting the theme.

Reply:
@devinbinnie I remember this discussion but then I saw that we can spread all object properties together here since we are already mutating the theme object. Let me know how you would approach this 👍